lectures/commod_price.md

```diff
@@ -138,7 +138,7 @@ We assume that the sequence $\{ Z_t \}_{t \geq 1}$ is {ref}`IID <iid-theorem>` w
 Speculators can store the commodity between periods, with $I_t$ units
 purchased in the current period yielding $\alpha I_t$ units in the next.
 
-Here $\alpha \in (0,1)$ is a depreciation rate for the commodity.
+In general, $\alpha$ is a factor. Here $\alpha \in (0,1)$ is a depreciation rate for the commodity.
```

Comment:
@SylviaZhaooo I am not sure what you mean by this addition. 

What about:

```
$\alpha$ is a variable $(0,1)$ which represents the depreciation rate for the commodity
```

Reply:
In Jiarui's comments #436, one suggestion was "Might consider treating as a more general factor rather than just simply the depreciation rate. Can potentially delete the phrase regarding depreciation", so I discussed with Jiarui just now. Perhaps we should say it is a parameter but not a factor?

Reply:
thanks @SylviaZhaooo what about

```
$\alpha$ is a parameter that that varies between $0 and $1$ and represents the depreciation rate for the commodity
```

Reply:
@SylviaZhaooo I have just seen that @jstac and reviewed and edited [053c529](https://github.com/QuantEcon/lecture-python-intro/pull/443/commits/053c529963cd334b3dec64c8ada49c2bc841ea48)